services/adgs/config/adgs_search_config.template.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2024 CS Group
```

Comment:
This is the template file for `adgs_search_config.yaml`. All common values are set only once in a root `template` block.